Sources/Basics/CMakeLists.txt

```diff
@@ -15,10 +15,13 @@ add_library(Basics
   FileSystem+Extensions.swift
   HTPClient+URLSession.swift
   HTTPClient.swift
-  JSON+Extensions.swift)
+  JSON+Extensions.swift
+  SwiftVersion.swift)
 target_link_libraries(Basics PUBLIC
   TSCBasic
   TSCUtility)
+target_link_libraries(Basics PRIVATE
+   TSCclibc)
```

Comment:
Because this is private (and imported using `@_implementationOnly`), it doesn't introduce a requirement on having the headers for TSCclibc around for any clients of libSwiftPM, right?  Getting rid of that was a good cleanup and we'd hate to give it up.

Reply:
yes, that was my plan - its the same setup as TSC has